### PR TITLE
Drop poetry and run CI tests again

### PR DESCRIPTION
**Pull request by jokester**
- much fewer people knows how to use it, compared to pip the Matured Technology
- current config using poetry is effectively unmaintained for more than 1y. leaving it as is only voids the meaning of CI 

closes https://github.com/zyddnys/manga-image-translator/issues/667

### CODE REVIEW

**Comment by jokester** [comment]
The pytest/lint checks are still breaking. Need help orz

**Comment by zyddnys** [comment]
is it ok that i merge this?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -1,46 +1,47 @@
...
+        python3 -mvenv venv
+        venv/bin/pip install -r requirements.txt -r requirements-dev.txt
+    - name: Test
+      run: |
+        venv/bin/pytest test
+      timeout-minutes: 5
+    - name: Lint *.py files
+      run: |
+        venv/bin/pylint $(git ls-files '*.py')
+      if: always()
```
Comment (jokester) [inline .github/workflows/ci.yml:46]: Now lint is always executed but lint error does not count as CI fail.

I couldn't fix the lint errors, there are too many of them 😨

**Comment by jokester** [comment]
@zyddnys yes please I think I did all I could.

Now the pytest check is running green. This should capture simple error like incorrect requirements.txt .

**Pull request merged**